pygtrie.py

```diff
@@ -1223,6 +1223,16 @@ def __init__(self, *args, **kwargs):
         self._separator = kwargs.pop('separator', '/')
         super(StringTrie, self).__init__(*args, **kwargs)
 
+    @property
+    def _separator(self):
+        return self.__separator
+
+    @_separator.setter
+    def _separator(self, sep):
```

Comment:
If you don't mind importing `sys` here I would prefer to add the proper type check with string_types copied from https://github.com/JioCloud/python-six/blob/master/six.py#L41
We come from py3 world and have some troubles with mixed `bytes` and `str`